CaloMC/fcl/common.fcl

```diff
@@ -1,5 +1,6 @@
 BEGIN_PROLOG
-  HitMakerBlindTime      : 500. 
+  HitMakerDigitizationStart    : 500. 
+  HitMakerDigitizationEnd      : 1705. 
```

Comment:
Is there a reason why it is 1705 and not a larger number (1775 if I kept the previous 80ns endBuffer)?

Reply:
The  StepPoints production job stops at 1705, which keeps the leading edge of the next proton pulses's flash, but cuts out the peak.    Even 50ns beyond that the rate is up by orders of magnitude, if we kept those the payload would be dominated by useless low-energy electron hits.    It doesn't make sense to digitize beyond 1705 as there are no detsteps to digitize.

Reply:
From my understanding of the code, it was never actually at 1775 because all the times folded at 1695 (although maybe I'm missing some details of the digitization process if this is a buffer to keep the end of a waveform or something). We don't actually keep any steps outside 1705 so going later than that doesn't buy you anything as far as getting later hits